app/services/map_marker_json.rb

```diff
@@ -1,9 +1,13 @@
 module MapMarkerJson
   def self.build(organisations)
-    Gmaps4rails.build_markers(Queries::Organisations.add_recently_updated_and_has_owner(organisations)) do |org, marker|
+    if organisations.first.is_a?(ActiveRecord::Base)
```

Comment:
checking for object type is usually a warning sign - we should look closely at seeing if we can avoid this